docs/2_4_common_schema.adoc

```diff
@@ -1927,7 +1927,24 @@ _Examples for `terminalKind`: `StandardXXX_Mechanical_Translational`, `Modelica.
 
 _The structured naming convention of the <<ModelVariables>> is independent from the terminal names and member variable names._
 
-_A tool may choose to connect terminals with a different or unknown `terminalKind`, if the `matchingRule` matches.]_
+_A tool may choose to connect terminals with a different or unknown `terminalKind`, if the `matchingRule` matches.
+
+
```

Comment:
```suggestion
A tool may choose to take the structured naming convention of variable names into account when matching terminals.
This can for example be used to allow matching between terminals employing arrays mapped to scalar variables using structured naming convention, and those using native FMI 3.0 arrays.
```